app/assets/javascripts/oxalis/controller.js

```diff
@@ -225,6 +227,22 @@ class Controller extends React.PureComponent {
           event.stopPropagation();
           Model.save();
         },
+
+        // In the long run this should probably live in a user script
+        "9": function toggleSegmentationOpacity() {
+          // Flow cannot infer the return type of getConfiguration :(
+          // Should be fixed once this is fixed: https://github.com/facebook/flow/issues/4513
+          const curSegAlpha = Number(api.data.getConfiguration("segmentationOpacity"));
+          let newSegAlpha = 0;
+
+          if (curSegAlpha > 0) {
```

Comment:
What happens if a user has the segmentation opacity set to 0, then starts a tracing and presses `9`? `prevSegAlpha` is undefined then and `newSegAlpha` will be set to undefined as well. Not sure what the api does if one sets undefined, but regardless I think `prevSegAlpha` should have a default value of 100. Do you agree?